tools/block-generator/generator/server.go

```diff
@@ -32,7 +33,7 @@ func MakeServer(configFile string, addr string) (*http.Server, Generator) {
 	noOp := func(next http.Handler) http.Handler {
 		return next
 	}
-	return MakeServerWithMiddleware(0, "", configFile, false, addr, noOp)
+	return MakeServerWithMiddleware(nil, 0, "", configFile, false, addr, noOp)
```

Comment:
Current log level is fine for the daemon?

Reply:
I don't think the daemon is really used for anything other than testing, so I didn't add anything special here.